bin/regular_mvn_build_deploy_analyze

```diff
@@ -103,6 +103,7 @@ elif [ "$TRAVIS_PULL_REQUEST" != "false" ] && [ -n "${GITHUB_TOKEN:-}" ]; then
       -Dsonar.analysis.prNumber=$TRAVIS_PULL_REQUEST \
       -Dsonar.branch.name=$TRAVIS_PULL_REQUEST_BRANCH \
       -Dsonar.branch.target=$TRAVIS_BRANCH \
+      -Dsonar.pullrequest.id=$TRAVIS_PULL_REQUEST \
       -Dsonar.pullrequest.github.id=$TRAVIS_PULL_REQUEST \
```

Comment:
why do you keep this setting?

Reply:
For a smoother upgrade path. Teams can update their build scripts anytime before we deploy the new `sonar-core-plugins`, and pull request decoration will still work, using the old way.

If I remove these lines, then teams would have to synchronize the updating of their build script with the deployment of `sonar-core-plugins`, otherwise they will miss pull request decoration. Which probably would not be the end of the world. But it's not too hard to make it easier for teams, so I prefer to remove the old settings in a second step.